ci: Clio docker image (#1509)
XRPLF/clio: docker/ci/README.md

# CI image for XRPLF/clio

This image contains an environment to build [Clio](https://github.com/XRPLF/clio), check code and documentation.

> It is used in [Clio Github Actions](https://github.com/XRPLF/clio/actions) and it could be used to compile Clio locally.

Reviewer: ```suggestion
It is used in [Clio Github Actions](https://github.com/XRPLF/clio/actions) but can also be used to compile Clio locally.
```
